corehq/apps/users/templates/users/enterprise_users.html

```diff
@@ -43,11 +43,22 @@ <h3 class="panel-title" style="padding-top: 7px;">{% trans 'Users' %}</h3>
              data-bind="visible: showUsers">
         <thead>
           <tr>
-            <th>{% trans "Username" %}</th>
-            <th>{% trans "Name" %}</th>
-            <th>{% trans "Role" %}</th>
+            <th >
+              <i id="sort-icon" class="clickable glyphicon glyphicon-sort" data-bind="attr: { name: 'username' }, event: { click: sortByColumn }"></i>
```

Comment:
Semi-nitpicks: I'd make `sort-icon` a class (the HTML spec says ids should be unique, which isn't a huge deal but will show warnings in the browser dev tools), use `data-name` instead of name (that's the typical way to add arbitrary data to elements), and make `data-name` a regular HTML attribute instead of setting it in knockout, since it isn't dynamic).

Reply:
Thanks, I'll make those changes.